### ISSUE TITLE
aiohttp removes query params passed to trace function

### ISSUE THREAD

**Issue by bharel** [fafhrd91/aiohttp#7722]
### Describe the bug

Sending a get request with embedded query params (session.get("foo.com/?a=1")) is supported, but when the url parameter is checked on the tracing function (TraceRequestStartParams.url) it is void of params ("foo.com").

The reason is [this line](https://github.com/aio-libs/aiohttp/blob/5a499d040654455eb03909ef6ab66a454316e317/aiohttp/client.py#L442C67-L442C67) which overrides the query params with the default "None" parameter set to the .get.

Fix will need to update (merge) the parameters instead of replace.

### To Reproduce

Create a session with a tracing function for start.
session.get("foo.com/?query=value")
check the url in the request.

### Expected behavior

Get the original URL.

### Logs/tracebacks

```python-traceback
-
```

### Python Version

```console
$ python --version
3.11.6
```

### aiohttp Version

```console
$ python -m pip show aiohttp
Name: aiohttp
Version: 3.8.6
Summary: Async http client/server framework (asyncio)
Home-page: https://github.com/aio-libs/aiohttp
Author: 
Author-email: 
License: Apache 2
Location: /Users/.../venv/lib/python3.11/site-packages
Requires: aiosignal, async-timeout, attrs, charset-normalizer, frozenlist, multidict, yarl
Required-by: aiobotocore
```

### multidict Version

```console
$ python -m pip show multidict
irrelevant
```

### yarl Version

```console
$ python -m pip show yarl
irrelevant
```

### OS

irrelevant

### Related component

Client

### Code of Conduct

- [X] I agree to follow the aio-libs Code of Conduct

**Comment by bharel** [comment]
Issue was fixed in the master version.

**Comment by bharel** [comment]
Duplicate of #7259
